settings.go

```diff
@@ -27,6 +27,30 @@ type Request struct {
 	PerPage typeutil.Undefined[int]
 }
 
+var (
+	// QueryParamSearch the name of search field in pagination
+	QueryParamSearch = "search"
+	// QueryParamFilter the name of filter field in pagination
+	QueryParamFilter = "filter"
+	// QueryParamOr the name of or field in pagination
+	QueryParamOr = "or"
+	// QueryParamSort the name of sort field in pagination
+	QueryParamSort = "sort"
+	// QueryParamJoin the name of join field in pagination
+	QueryParamJoin = "join"
+	// QueryParamFields the name of fields field in pagination
+	QueryParamFields = "fields"
+	// QueryParamPage the name of current page index in pagination
+	QueryParamPage = "page"
+	// QueryParamPerPage the name of the data size field for each page in pagination
+	QueryParamPerPage = "per_page"
+	// DefaultPageSize the default pagination page size if the "per_page" query param
+	// isn't provided.
+	DefaultPageSize = 10
+
+	modelCache = &sync.Map{}
+)
+
 // NewRequest creates a filter request from an HTTP request's query.
 // Uses the following entries in the query, expected to be validated:
```

Comment:
```suggestion
// Uses the entries defined by the "QueryParam*" global variables from the given query. All those entries are expected to be validated.
```

The list below is not necessary anymore.